README.md

```diff
@@ -22,10 +22,20 @@ const MyComponent = () => <CloseSVG />;
 
 ## Installation
 
+You only need to install this package as a dev dependency because it's only needed during compilation.
+
+For NPM:
+
+```
+npm install -D babel-plugin-inline-react-svg
 ```
-npm install --save-dev babel-plugin-inline-react-svg
+
+For Yarn:
+```
+yarn add -D babel-plugin-inline-react-svg
```

Comment:
why is this needed? yarn users always have to know how to translate `npm` commands into `yarn` commands, or they won't be able to interact with most of the ecosystem.